### ISSUE TITLE
Feature request: update to latest pdf.js v2.7.570

### ISSUE THREAD

**Issue by igorlino** [VadimDez/ng2-pdf-viewer#714]
##### Bug Report or Feature Request (mark with an `x`)
```
- [ ] Regression (a behavior that used to work and stopped working in a new release)
- [ ] Bug report -> please search issues before submitting
- [ ] Feature request
- [ ] Documentation issue or request
```

https://github.com/mozilla/pdf.js/releases

pdf viewer does not contain the last 3 minor releases:
v2.7.570 …
 18 hours ago  f2c7338  zip  tar.gz  Notes  https://github.com/mozilla/pdf.js/releases/tag/v2.7.570
v2.6.347 …
 on 3 Sep 2020  3be9c65  zip  tar.gz  Notes  https://github.com/mozilla/pdf.js/releases/tag/v2.6.347
v2.5.207 …
 on 1 Jun 2020  0974d60  zip  tar.gz  Notes  https://github.com/mozilla/pdf.js/releases/tag/v2.5.207

Perhaps is a good time to update to v2.7.570 that includes several performance improvements.

**Comment by cvjanick** [comment]
Also if going to v2.7.570 please use the minified versions of es2015 or later or provide option to do so for smaller code and better performance. See the nice but somewhat bloated ngx-extended-pdf-viewer for some examples, going minified might make this lightweight ng2-pdf-viewer library even better.

**Comment by igorlino** [comment]
@cvjanick when the pdf-js npm dependency is node_modules/pdfjs-dist/build, you have the minified versions available there
I think that package is only a dependency and Vadim cannot change anything about it.

At the moment I have 2 changes in my project to make things work:
1)
in angular.json
![image](https://user-images.githubusercontent.com/10480442/105956819-a8d7f780-6078-11eb-9bf1-4cbd24c057b8.png)

2)
```
        // Set custom path to the worker to avoid loading of CDN
        if (environment.production) {
            (window as any).pdfWorkerSrc = '/assets/pdf.worker.min.js';
        }

```

**Comment by cvjanick** [comment]
Thank you igorlino will give that a try.

**Comment by cvjanick** [comment]
Also thanks to both Stephan and Vadim - two great libraries! Using both for different purposes.

**Comment by magorismagor** [comment]
+1 to this Feature Request!

**Comment by igorlino** [comment]
@VadimDez sorry to spam, in the meantime there are 6 releases, not sure if it is hard to integrate to the latest one, but each release seems to contain a very significant amount of features/bug fixes/performance improvements.

v2.10.377 …
 on Jul 25  156762c  
v2.9.359 …
 on May 30  e667c8c 
v2.8.335 …
 on Apr 5  228adbf  
v2.7.570 …
 on Jan 24  f2c7338
v2.6.347 …
 on Sep 3, 2020  3be9c65 
v2.5.207 …
 on Jun 1, 2020  0974d60 
 
 
https://github.com/mozilla/pdf.js/releases

**Comment by igorlino** [comment]
sorry my bad, there you did already a release in https://github.com/VadimDez/ng2-pdf-viewer/pull/777, so technically you covered already the reported releases. 
Please consider if a new ticket is needed for the next 3 releases, or this ticket can be retaken to cover once again

**Comment by beckedorf** [comment]
FYI I created a quick, temporary fix, if you want to use it:

https://github.com/beckedorf/ng2-pdf-viewer
https://www.npmjs.com/package/pdf-forms-viewer-temp

**Comment by VadimDez** [comment]
Closed by #829
